irohad/model/generators/impl/block_generator.cpp

```diff
@@ -23,9 +23,8 @@ namespace iroha {
     namespace generators {
       Block BlockGenerator::generateGenesisBlock(
           std::vector<std::string> peers_address) {
-        Block block;
-        block.created_ts =
-            (ts64_t)std::chrono::system_clock::now().time_since_epoch().count();
+        Block block{};
```

Comment:
What is the reason to set curly brackets?

Reply:
It default-initializes the fields of a struct.